From 61f1a73437f598f3f4649fc3d48fecfcb8e91de5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?G=C3=BCnther=20Deschner?= <gd@samba.org>
Date: Thu, 28 Apr 2016 12:58:33 +0200
Subject: [PATCH 1022/1096] s3:librpc:crypto:gse: increase debug level for
 gse_init_client().

BUG: https://bugzilla.samba.org/show_bug.cgi?id=11872

Guenther

Signed-off-by: Guenther Deschner <gd@samba.org>
Reviewed-by: Stefan Metzmacher <metze@samba.org>
Reviewed-by: Andreas Schneider <asn@samba.org>
(cherry picked from commit b6595037f3fcaafb957d9c08edfb89c72cded987)
(cherry picked from commit 9aa4b3c5d86a88d8b73d5bfba0d67a1cc05cd16d)
---
 source3/librpc/crypto/gse.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source3/librpc/crypto/gse.c b/source3/librpc/crypto/gse.c
index ffdfde1..963c98a 100644
--- a/source3/librpc/crypto/gse.c
+++ b/source3/librpc/crypto/gse.c
@@ -239,7 +239,7 @@ static NTSTATUS gse_init_client(TALLOC_CTX *mem_ctx,
 				  GSS_C_NT_USER_NAME,
 				  &gse_ctx->server_name);
 	if (gss_maj) {
-		DEBUG(0, ("gss_import_name failed for %s, with [%s]\n",
+		DEBUG(5, ("gss_import_name failed for %s, with [%s]\n",
 			  (char *)name_buffer.value,
 			  gse_errstr(gse_ctx, gss_maj, gss_min)));
 		status = NT_STATUS_INTERNAL_ERROR;
-- 
2.9.0

